### PR TITLE
Integrate project harfbuzz with target hb-subset-fuzzer

### PR DESCRIPTION
**Pull request by Vaush**

### CODE REVIEW

File: benchmarks/harfbuzz_hb-subset-fuzzer/hb-draw-fuzzer.options [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+timeout = 5
```
Comment (inferno-chromium) [inline benchmarks/harfbuzz_hb-subset-fuzzer/hb-draw-fuzzer.options:2]: Remove all other option files other than subset fuzzer.

File: benchmarks/harfbuzz_hb-subset-fuzzer/build.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# Build the library.
+meson --default-library=static --wrap-mode=nodownload \
+      -Dexperimental_api=true \
+      -Dfuzzer_ldflags="$(echo $LIB_FUZZING_ENGINE)" \
+      $build \
+   || (cat build/meson-logs/meson-log.txt && false)
+
+# Build the fuzzers.
+ninja -v -j$(nproc) -C $build test/fuzzing/hb-{shape,draw,subset,set}-fuzzer
+mv $build/test/fuzzing/hb-{shape,draw,subset,set}-fuzzer $OUT/
```
Comment (inferno-chromium) [inline benchmarks/harfbuzz_hb-subset-fuzzer/build.sh:39]: remove everything other than subset fuzzer in this build.sh files.

**Review by inferno-chromium — CHANGES_REQUESTED** [review]

**Comment by jonathanmetzman** [comment]
I think you need to blocklist klee.

**Comment by Vaush** [comment]
> 
> 
> I think you need to blocklist klee.

I will blocklist it, but apparently the problem is that klee doesn't support emtpy seeds (at least, that's what the error says). Still, I don't think deleting part of the corpus would be a good idea.

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**